conf/application.conf

```diff
@@ -349,11 +349,7 @@ db {
   #default.logSql=true
 }
 
-pidfile.path = "/dev/null"
-
-app.portWebapp = 8085
-
-webapi.port = 8080
-webapi.host = "http://localhost"
-instance.registry.path = "http://localhost:8087"
-play.modules.enabled += "app.EagerLoaderModule"
\ No newline at end of file
+play.filters.hosts {
+  # Allow requests to localhost:9000.
+  allowed=["localhost:9000","127.0.0.1:9000","pg-delphi.cs.uni-paderborn.de:8080"]
```

Comment:
Again... Production configuration inside of the source code... Please remove this or replace by something configurable from the outside. (cf. DELPHI_WEBAPI_URL environment variable.)

Reply:
I have removed it for now, we'll use it as env variable in future. Thanks!

Reply:
I'll keep this comment open as a reminder.

Reply:
It has been updated with an env variable. Please check whether I did it right or not. Thanks!

Reply:
It's better now. Thanks.